src/Module/CreateCommand.php

```diff
@@ -51,7 +54,8 @@ protected function execute(InputInterface $input, OutputInterface $output) : int
     {
         $module = $input->getArgument('module');
         $composer = $input->getOption('composer') ?: 'composer';
-        $modulesPath = CommandCommonOptions::getModulesPath($input);
+        $config = $this->getConfig(realpath(getcwd()));
```

Comment:
This is the line that is leading to the error you're seeing.

The other tests against code that composes that trait will create a virtual filesystem and inject that as the project root into the given class, allowing this to work correctly. You can see an example of that in the `ZendTest\Expressive\Tooling\CreateHandler\CreateTemplateTest` class.

Since you're only using the `getConfig()` method (and, by extension, `getContainer()` method), I'd argue those should be split into their own trait, which the `TemplateResolutionTrait` will then compose. `CreateCommand` can then compose only the new trait. Either way, testing against it will be the same.

Reply:
Thanks for the help. A separate trait makes sense, I extracted `getConfig()` and `getContainer()`. I am not sure, if `ConfigAndContainerTrait` is the best...

Test have been fixed.

Reply:
Name of the new trait totally works for me; describes exactly what it provides.